packages/lib/src/components/types.ts

```diff
@@ -81,13 +81,19 @@ export type UIElementStatus = 'ready' | 'loading' | 'error' | 'success';
 
 export type PayButtonFunctionProps = Omit<PayButtonProps, 'amount'>;
 
+export interface ActionHandledReturnObject {
+    componentType: string;
+    actionType: string;
```

Comment:
Maybe we could define the set of possible values here for the `actionType` ? 
It will make more clear for who is implementing this callback to know what to expect from the `actionType` property

Ex:
```ts
actionType: 'qr-code-loaded' | 'polling-started' ...
```

Reply:
Changed `actionType` to `actionDescription` and give it a type (`ActionDescriptionType`)